### DISCUSSION TITLE
Text lengh in Table Block

### DISCUSSION THREAD

**Discussion by Sagestum** [Budibase/budibase#11053] [Help] [answered]
Hello everyone, 

How can I increase the text length in a table block?

After about 35 characters the output of a text is stopped and replaced with dots. Like "Well, hello there..." 

![image](https://github.com/Budibase/budibase/assets/11883823/4fb55abe-4efe-457a-8262-034e1b30e8b6)

Is it possible to increase the text length before the ... replaces the text?
The size of the table does not seem to affect this. 

thanks 

best
Chris

**Comment by ConorWebb96** [comment, marked as answer]
Hey @Sagestum,

Currently I don't believe there is a way for this to be increased. You can create an feature request here if you would like this to be added to the core product; https://github.com/Budibase/budibase/issues/new/choose

**Sidenote**, I believe there is a custom plugin called super table which can be installed and used to do what you are want. This can be found here; https://github.com/poirazis/bb-component-SuperTable

I hope this helps.

**Reply by Sagestum** [reply]
I have created a feature request: https://github.com/Budibase/budibase/issues/11054

**Comment by Sagestum** [comment]
Yes, I have seen SuperTable, but I have tried to use as few plugins as possible. Although SuperTable looks a bit overkill, but well, I guess I have to try it now.

**Reply by andz-bb** [reply]
hey @Sagestum,

as a quick workaround you can set a fixed column width for the text column that you'd like to see more of. In the table block, click _Configure columns_ and then _Add all columns_. Click the settings cog for the column you'd like to adjust and then provide a fixed width. This stops the table from dynamically sizing the columns so it may take some adjustment to get it to display properly, but once you have set the width it will be able to display a lot more of your text.

Here's how mine looks initially, just like your screenshot above:
![auto column](https://github.com/Budibase/budibase/assets/110921612/f4b92f83-2f80-42c1-b1aa-5b330921d914)

and now after setting the Haupttext column to be fixed to 1080px wide:
![wider column](https://github.com/Budibase/budibase/assets/110921612/b1a28472-652c-4ece-b7d1-e1a443b2623a)
